spec/reql_spec.cr

```diff
@@ -4,27 +4,37 @@ r.db("test").table_list.for_each do |name|
-  {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mod.yaml") }}
-  {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mul.yaml") }}
-  {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/sub.yaml") }}
-  {{ run("./reql_spec_generator", "spec/rql_test/src/mutation/update.yaml") }}
-  {{ run("./reql_spec_generator", "spec/rql_test/src/range.yaml") }}
-  {{ run("./reql_spec_generator", "spec/rql_test/src/selection.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/aggregation.yaml") }}
+  {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/edge.yaml") }}
+  
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/geo.rb.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/idxcopy.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/include_states.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/point.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/sindex.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/squash.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/changefeeds/table.yaml") }}
+  
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/control.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/array.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/bool.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/null.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/number.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/object.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/string.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/typeof.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/datum/uuid.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/default.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/add.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/aliases.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/div.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/floor_ceil_round.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/logic.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/math.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mod.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mul.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/sub.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/mutation/update.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/range.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/selection.yaml") }}
```

Comment:
Any reason for commenting these tests out?

Reply:
Not needed in this PR.
I start ading more features not handled yet in the drivers but handled in tests.
I'll revert that for this PR.